t/shdict-simple.t

```diff
@@ -119,6 +119,8 @@ true
 true
 --- error_log
 try to set key 1th time
+--- no_error_log
```

Comment:
Better to use `grep_error_log` instead. It's more stringent.

Reply:
fixed